### PR TITLE
[Ubuntu] Remove zip versions from Azuremodules

### PR DESCRIPTION
**Pull request by sssharif**
Removed zip versions redundant code from PowerShellAZ module script and removed the properties from toolsets.json.

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/ubuntu/toolsets/toolset-2004.json [hunk 1]
```diff
@@ -109,8 +109,6 @@
             "url" : "https://raw.githubusercontent.com/Azure/az-ps-module-versions/main/versions-manifest.json",
```
Comment (mikhailkoliada) [inline images/ubuntu/toolsets/toolset-2004.json:109]: do we still need that url?

File: images/ubuntu/scripts/build/Install-PowerShellAzModules.ps1 [hunk 2]
```diff
@@ -24,20 +24,6 @@ foreach ($module in $modules) {
 
     $assets = Invoke-RestMethod $module.url
```
Comment (mikhailkoliada) [inline images/ubuntu/scripts/build/Install-PowerShellAzModules.ps1:25]: we do not use this anymore I guess

**Review by maxim-lobanov — APPROVED** [review]

**Pull request merged**